### ISSUE TITLE
Review and fix web accessibility errors, alerts

### ISSUE THREAD

**Issue by libcce** [LibraryCarpentry/librarycarpentry.github.io#4]
Review and fix web accessibility errors, alerts via WAVE tool:
http://wave.webaim.org/report#/https://librarycarpentry.org

**Comment by jscaffrey** [comment]
Please consider these changes to remediate WAVE errors and alerts. In line with your mission, this is my first pull request ever. Thanks for being an awesome community, and I hope to work with you in the future.

WAVE report before these changes:  5 errors, 16 alerts, 2 contrast errors
<img width="807" alt="" src="https://user-images.githubusercontent.com/10285039/51092395-aa217a80-1764-11e9-8d08-2478a95892d2.png">

WAVE report after these changes: 0 errors, 8 alerts, 0 contrast errors
<img width="591" alt="" src="https://user-images.githubusercontent.com/10285039/51092400-b7d70000-1764-11e9-8621-0dfc638af543.png">

-Julia Caffrey
Web Services Librarian, TU

**Comment by tracykteal** [comment]
Thank you so much! These are important changes and really appreciate your work on this. I see your issue update here, but not a PR yet. I just wanted to make sure we hadn't missed it.

**Comment by libcce** [comment]
Thanks @jscaffrey I merged your pull request but then had to revert it https://github.com/LibraryCarpentry/librarycarpentry.github.io/pull/22 as the header image for the site (library shelves) wasn't appearing and other fonts were not displaying correctly.

**Comment by katrinleinweber** [comment]
Did it behave differently offline? I think this can be debugged locally with `git checkout 8d968e37b02c2ce41efaeed98e4c62fb8e1a22c2 && make serve`.

**Comment by jscaffrey** [comment]
@katrinleinweber Thanks. Here's what I see after running that, so unable to replicate issue reported by @libcce . Header image and all fonts rendering on my end.

![screencapture-localhost-4000-2019-01-27-09_04_25](https://user-images.githubusercontent.com/10285039/51801966-b4d31980-2212-11e9-8cf6-d562ba123f1e.png)

**Comment by libcce** [comment]
Hi @jscaffrey @katrinleinweber I've run WAVE http://wave.webaim.org/report#/https://librarycarpentry.org and we now have 1 error which is great but our alerts are up to 8 and there are 34 contrast errors. The contrast errors have me more concerned at the moment.

**Comment by jscaffrey** [comment]
Hi @libcce ! Thanks for following up. The contrast errors are due to:

1. The colors used on the menu  when you're hovering over the option (light pink on light gray). Can be fixed by updating the scss files and regenerating.
2. Lack of fallback color on the header image. Can be fixed at the same time as the remaining accessibility error by moving away from the animated header image, killing two birds with one stone so to speak.

I can put in a PR for these but want to note that if you are testing with `make site`  or `jekyll build` the CSS will link to the live CSS and not the new/rebuilt CSS which will cause the header to not show up and possibly other issues. I recommend testing with `make serve` or `jekyll serve`

**Comment by maneesha** [comment]
Assigning to @kariljordan to consider as part of the The Carpentries Equity, Inclusion, and Accessibility Roadmap

**Comment by kariljordan** [comment]
Thank you for bringing this to my attention. I have a "save for later" project that looks at web accessibility for all of our websites, but that project will more than likely commence next year some time. Please keep this issue open, and if there is a group of community members interested in working on web accessibility send them my way!

**Comment by libcce** [comment]
@kariljordan there is another related issue https://github.com/LibraryCarpentry/librarycarpentry.github.io/issues/39 which could impact how we are listed in Google searches. I believe it is across Carpentries sites.

**Comment by maneesha** [comment]
Assigning this to @brynnelliott - not sure what your workflow or workload is like, so prioritize as you see fit

**Comment by maneesha** [comment]
Closing as we are now using a new theme and design.
